src/commands/txCommands/util/poFormatters.js

```diff
@@ -51,19 +51,33 @@ const poObjToPoString = poObj => {
 };
 
 // Array<MessageDescriptor> => Po
+// https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html
+/**
+ * special handling of description object
+ * - 'text' will be assigned to `comments.extracted`.
+ * - all other key/values will be converted to a 'key: value; ...' string and
+ *   assigned to `comments.translator`, e.g.
+ *
+ * description = {
+ * 	text: 'info for translator',
+ *  jira: 'asdfas',
```

Comment:
Not sure if there are other pages, but looks like this documentation could use updating - https://meetup.atlassian.net/wiki/spaces/WEG/pages/714178689/How+to+add+translations+to+your+component.

Also, the `i18n` link in the `mup-web` README is broken. Perhaps it should link to that or some other confluence pages?